controllers/tlspolicies_validator.go

```diff
@@ -99,6 +104,22 @@ func (t *TLSPoliciesValidator) isTargetRefsFound(topology *machinery.Topology, p
 	return nil
 }
 
+// isConflict Validates if there's already an older policy with the same target ref
+func (t *TLSPoliciesValidator) isConflict(policies []machinery.Policy, p *kuadrantv1.TLSPolicy) error {
+	conflictingP, ok := lo.Find(policies, func(item machinery.Policy) bool {
+		conflictTLSPolicy := item.(*kuadrantv1.TLSPolicy)
+		return p != conflictTLSPolicy && conflictTLSPolicy.DeletionTimestamp == nil &&
+			conflictTLSPolicy.CreationTimestamp.Before(&p.CreationTimestamp) &&
+			reflect.DeepEqual(conflictTLSPolicy.GetTargetRefs(), p.GetTargetRefs())
+	})
+
+	if ok {
+		return kuadrant.NewErrConflict(kuadrantv1.TLSPolicyGroupKind.Kind, client.ObjectKeyFromObject(conflictingP.(*kuadrantv1.TLSPolicy)).String(), errors.New("conflicting policy"))
+	}
+
+	return nil
+}
+
```

Comment:
@mikenairn This is what I'd done if you want to re-introduce `conflict` for DNSPolicy also 🤔